### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by kevalmehta17** [open-sauced/feedback#45] [Q&A]
I got student developer pack, im excited about open Source but never did before!
so how can i make my first Pull request?
i have MERN knowledge but never did opensource contribution!!
If anyone can suggest me that help a lot in my career!! 
Thank you!! :)

**Comment by imDarshanGK** [comment]
Hi @kevalmehta17,
To make your first pull request:

* Find a project with "Good first issue" or beginner-friendly tasks.
* Fork the repo (copy it to your account).
* Clone it to your computer.
* Make changes (fix a bug, update docs, etc.).
* Commit your changes.
* Create a Pull Request to the original repo.
